code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -108,18 +108,23 @@
 	var/agent_number = 1
 	var/spawnpos = 1
 
+	var/obj/machinery/nuclearbomb/syndicate/the_bomb
+	if(nuke_spawn && synd_spawn.len > 0)
```

Comment:
`length()` is better than `len` generally, and you can remove the `> 0` since it's implied.
```suggestion
	if(nuke_spawn && length(synd_spawn))
```
_(I realise this was just moved over, but no reason not to clean it up a bit too)_

Reply:
Do this change